plugins/Kaleidoscope-OneShot/src/kaleidoscope/plugin/OneShot.h

```diff
@@ -201,13 +210,22 @@ class OneShot : public kaleidoscope::Plugin {
 
   // --------------------------------------------------------------------------
   // Configuration variables
-  uint16_t timeout_           = 2500;
-  uint16_t hold_timeout_      = 250;
-  int16_t double_tap_timeout_ = -1;
-
-  uint16_t stickable_keys_ = -1;
-  bool auto_modifiers_     = false;
-  bool auto_layers_        = false;
+  struct Settings {
+    uint16_t timeout;
+    uint16_t hold_timeout;
+    int16_t double_tap_timeout;
+
+    uint16_t stickable_keys;
+    bool auto_modifiers;
+    bool auto_layers;
+  } settings_ = {
```

Comment:
I think it would be better to put the default values in the `Settings` struct definition, and use a separate statement to declare the `Settings settings_;` member, given the fragility of initializing values this way.  See MouseKeys for reference.